pyproject.toml

```diff
@@ -64,7 +64,7 @@ zstandard = ">=0.13.0,<1.0.0"
 tenacity = ">=8.2.3,<10.0.0"
 pyarrow = { version = ">=14.0.0,<18.0.0", optional = true }
 pandas = { version = ">=1.0.0,<3.0.0", optional = true }
-duckdb = { version = ">=0.5.0,<2.0.0", optional = true }
+duckdb = {version = "1.1.0", optional = true}
```

Comment:
I'm not sure how I feel about pinning this to a single version 🤔 

Is this necessary to resolve the issues with our CI, given that the new version is out, and our upper limit is already <2.0.0?

Reply:
Good point, not sure about this one. The CI issue can only be resolved by v1.1.0 or higher. I'm not sure if CI will automatically pick the newest version of the library.

Reply:
Let's see if its necessary https://github.com/kevinjqliu/iceberg-python/pull/4

v1.1.0 is already the official version on pypi 
https://pypi.org/project/duckdb/#history

Reply:
Looks like that [didn't work](https://github.com/kevinjqliu/iceberg-python/actions/runs/10781147671/job/29898441212?pr=4), its still installing duckdb v1.0.0